docs/examples/201/policy-definition-with-assignment/main.bicep

```diff
@@ -0,0 +1,77 @@
+targetScope = 'subscription'
+
+param listofAllowedLocations array {

```

Comment:
Can we use camelCasing? so `listofAllowedLocations` changes to `listOfAllowedLocations`